tests/test_dse.py

```diff
@@ -57,7 +57,16 @@ def test_scheduling_after_rewrite():
       'Eq(tu, ((ti0*ti1*t0)*tv + (ti0*ti1*tv)*t1))'],
      ['t0 + 2.0', 't0*ti0[x, y, z]*ti1[x, y, z]', 't1*ti0[x, y, z]*ti1[x, y, z]']),
 ])
-def test_yreplace_time_invariants(tu, tv, tw, ti0, ti1, t0, t1, exprs, expected):
+def test_yreplace_time_invariants(exprs, expected):
+    grid = Grid((3, 3, 3))
+    dims = grid.dimensions
+    tu = TimeFunction(name="tu", grid=grid, space_order=4).indexify()
```

Comment:
I'm writing it here, but basically this comment applies to all your edits -- I don't actually think we need the `.indexify()` ? have you tried without? 
When doing this sort of cleanup, I never needed it

Reply:
Object are indexed in the expressions to evaluate so yeah it breaks without it, had to go back to figure out why the test where failing after the changes and put it back.

Reply:
ahh... I think what other tests do is to use parentheses `(x, y, z)` instead of index notation in the expressions